align/cmdline.py

```diff
@@ -87,6 +87,9 @@ def __init__(self, *args, **kwargs):
         parser.add_argument('--version',
                             action='version',
                             version='%(prog)s ' + __version__)
+#        parser.add_argument('--python_gds_json',
```

Comment:
@parijatm Should we just always run this? Or have an argument to disable it.

Reply:
I think we should just generate it by default.